src/Message/PxFusionPurchaseRequest.php

```diff
@@ -11,8 +11,8 @@
  */
 class PxFusionPurchaseRequest extends AbstractRequest
 {
-    protected $endpoint = 'https://sec.paymentexpress.com/pxf/pxf.svc';
-    protected $namespace = 'http://paymentexpress.com';
+    protected $endpoint = 'https://sec.windcave.com/pxf/pxf.svc';
+    protected $namespace = 'http://windcave.com';
```

Comment:
The endpoint is updated, but the namespace is still paymentexpress.com. See https://github.com/thephpleague/omnipay-paymentexpress/issues/64

Reply:
Solved on #65